sidebars.js

```diff
@@ -851,6 +852,841 @@ const sidebars = {
       ],
     },
   ],
+
+  // Japanese sidebar nav
+  docsJapanese: [
+    {
+      type: 'doc',
+      label: 'ScalarDB ドキュメンテーションホーム',
+      id: 'index',
+    },
+    {
+      type: 'category',
+      label: 'ScalarDB について',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'overview',
+          label: '概要',
+        },
+        {
+          type: 'doc',
+          id: 'design',
+          label: 'デザイン',
+        },
+        {
+          type: 'doc',
+          id: 'requirements',
+          label: '要件',
+        },
+        {
+          type: 'doc',
+          id: 'database-configurations',
+          label: '基盤となるデータベースの設定',
```

Comment:
The term `基盤となるデータベース` would be hard for ScalarDB beginners to understand. I think `設定` or `基本の設定` would be sufficient because it can contain the database configurations as well.

Reply:
Looks good. Thanks!